lib/src/cancelable_operation.dart

```diff
@@ -202,7 +209,9 @@ class CancelableCompleter<T> {
     });
   }
 
-  /// Completes [operation] to [error].
+  /// Completes [operation] with [error].
```

Comment:
Doesn't say that it sets `isCompleted` to true, but refers to the value of `isCompleted` instead of whether the completer has been completed.
Prefer to speak in logical terms ("completer has been completed") instead of physical terms ("`isCompleted` is true"). The latter is a consequence of being completed, but then you also have to specify when `isCompleted` becomes true in precise detail.

Instead:
```dart
/// This method may not be called after either [complete] or [completeError] 
/// has been called once.
/// The [isCompleted] is true when either of these methods have been called.
```

Ditto for `completeError`.